docs/_data/table/en/user-configuration.yml

```diff
@@ -99,11 +99,12 @@ alluxio.user.short.circuit.enabled:
 alluxio.user.ufs.block.open.timeout.ms:
   N/A
 alluxio.user.ufs.block.read.concurrency.max:
-  N/A
+  The maximum concurrent readers for one UFS block on one Block Worker.
```

Comment:
I think there are only 3 properties updated? I've added the descriptions but I didn't regenerate the docs due to the issue you plan to fix.